hazelcast/src/main/java/com/hazelcast/config/SerializationConfig.java

```diff
@@ -51,6 +51,7 @@
     private boolean allowUnsafe;
     private final Set<ClassDefinition> classDefinitions;
     private JavaSerializationFilterConfig javaSerializationFilterConfig;
+    private boolean allowOverrideDefaultSerializers;
```

Comment:
Can we put field to equals and hashcode of SerializationConfig?

Reply:
fixed